group_vars/wingedhelix_cluster/vars.yml

```diff
@@ -96,6 +96,13 @@ external_jumphosts:
       - hostname: cf-porch
         network: external
```

Comment:
```suggestion
        network: external
      - hostname: az-porch
        network: az_public
```

Reply:
The azure jumphost is defined on BB, but is missing from the CF and WH vars. Therefore they are also missing from 
the `group_vars/[CF and WH]/ssh_client_settings.yml`